opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -1045,7 +1046,7 @@ def start_as_current_span(
         context: Optional[context_api.Context] = None,
         kind: trace_api.SpanKind = trace_api.SpanKind.INTERNAL,
         attributes: types.Attributes = None,
-        links: Sequence[trace_api.Link] = (),
+        links: Optional[Sequence[trace_api.Link]] = (),
```

Comment:
Do you really need Optional for empty tuples?

Reply:
this is mostly for the compatibility with the parent class:

`trace_api.Tracer` uses `_Links` 
https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-api/src/opentelemetry/trace/__init__.py#L335C16-L335C22

Which is defined as `Optional[Sequence[Link]]`:
https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-api/src/opentelemetry/trace/__init__.py#L153